### DISCUSSION TITLE
如何实现使用分包作为组件库

### DISCUSSION THREAD

**Discussion by lovelyqun** [NervJS/taro#13446] [Q & A]
现在的问题:
所有的业务页面都依赖字体图标库(500kb),和echarts(800kb)两个库,所以现在这两个库是在app.js中引入,自然会打包到主包了;
问题是主包超过了2M大小,没法上传了.
思路:
现在想把这两个库单独打包到一个子包中,然后在主包中引入这个子包.

现在的问题是我要怎么配置打包选项,才能把这两个库单独打包到子包的目录中.

**Comment by TheKonka** [comment]
> 现在想把这两个库单独打包到一个子包中,然后在主包中引入这个子包.

这个不就是直接打包到主包吗？小程序支持你说的这种主包引入子包吗

**Comment by yexz77** [comment]
这个问题解决了吗
